llvm/test/CodeGen/AMDGPU/GlobalISel/madmix-constant-bus-violation.mir

```diff
@@ -0,0 +1,110 @@
+# NOTE: Assertions have been autogenerated by utils/update_mir_test_checks.py UTC_ARGS: --version 5
```

Comment:
Is this test case `llvm-reduce`d already?

Reply:
Now it is. :-)